static_src/components/disclaimer.jsx

```diff
@@ -15,20 +15,52 @@ export default class Disclaimer extends React.Component {
 
   render() {
     let flagImg;
+    let dotGovIconImg;
+    let httpsIconImg;
+    const dotGovIcon = require('cloudgov-style/img/icon-dot-gov.svg');
+    const dotGovAlt = 'Dot gov';
+    dotGovIconImg = <img className={ this.styler('usa-banner-icon','usa-media_block-img') } src={ dotGovAlt } src={ dotGovIcon }</img>;
```

Comment:
I think we can just put this image tag inline, no need to save it in a variable.